docs/staking-strategies.md

```diff
@@ -4,15 +4,18 @@ title: Strategies
 sidebar_label: Strategies
 ---
 
-## Conservative
+## Liquidation risk
+A key factor to keep in mind when formulating a staking strategy is the risk of liquidation. As described in <a href="https://blog.synthetix.io/liquidation-faqs/" class="link">the liquidation blog post</a> and <a href="https://sips.synthetix.io/sccp/sccp-25" class="link">this liquidation proposal</a>, your stake is at risk of liquidation if your collateral ratio (`C-RATIO`) has gone under 200%, and remained less than 500% for more than three days. You should aim to keep your `C-RATIO` well above that, and check it at least every three days. Note that these thresholds are subject to change.
+
+## Conservative Strategy
 A conservative staker doesn't mint the maximum amount of sUSD, but instead targets an initial `C-RATIO` of 800%-900%. If the value of SNX increases and their ratio moves significantly above 900%, the conservative staker might consider minting more sUSD to restore the target ratio range. This staker will use sUSD minted to mirror the global synth portfolio on <a href="https://dashboard.synthetix.io/" class="link" target="_blank">Synthetix Dashboard</a>.
 
-## Neutral
+## Neutral Strategy
 A neutral staker targets an initial `C-RATIO` of 725%-775%. If the value of SNX increases and their ratio moves well above 725%, the neutral staker would consider minting more sUSD to restore a ratio within the target range. This staker might exchange sUSD for ETH, loan out sUSD, or trade for other synthetics, but more likely they would try to align with the global synth portfolio.
 
 
-## Aggressive
```

Comment:
i think you mean "bring their ratio back above 500%"

Reply:
Are you talking about the descriptions of the strategies themselves? I don't feel qualified to adjust those; my intention with this PR was to add a note about liquidation risk.

Reply:
well, there's no reason to burn above 600%. you may not want to mint below that number but once you have, there's no great reason to burn above 600%, only 500% which is target c-ratio right now

Reply:
OK, can I leave those details to someone else, though? I'm worried I'll mess them up because I'm inexperienced with SNX. I'd be happy to add a note saying that the staking thresholds are subject to change, and readers should check them when formulating their own staking strategies.

Reply:
no worries, fixed nums for you. thanks for contributing!

Reply:
Thanks!